uwsgi.ini

```diff
@@ -6,7 +6,7 @@
 # https://www.techatbloomberg.com/blog/configuring-uwsgi-production-deployment/
 socket = /tmp/kolibri_uwsgi.sock
 chmod-socket = 660
-gid = www-data
+chown-socket = $(KOLIBRI_USER):www-data
```

Comment:
I think gid should be kept, uwsgi might be using it for something else than for the socket.
maybe adding
`uid = $(KOLIBRI_USER)`
instead is a better solution (in case env variables are used by uwsgi when reading its ini file, I am not sure that's even happening)

Reply:
Before, it was set to make Nginx able to communicate with the socket. But since this also affected log ownership, I switched to `chown-socket` which only affects the socket, and not the logs.